examples/format/imperative/.expected/diff.patch

```diff
@@ -1,37 +1,40 @@
 diff --git a/resources.yaml b/resources.yaml
-index 968a0f4..a33d07a 100644
+index 2ce8de6..6c12c43 100644
 --- a/resources.yaml
 +++ b/resources.yaml
-@@ -1,22 +1,22 @@
+@@ -2,24 +2,24 @@
  apiVersion: apps/v1
  kind: Deployment
  metadata:
 -  labels:
 -    foo: bar
--    env: dev
+-    env: dev # must be one of prod, staging and dev
    name: my-nginx
+   # namespace for team foo
    namespace: my-space
 +  labels:
```

Comment:
can you explain the formatting changes in the PR description.

Reply:
This diff is from the `format` function. The formatting change is desired.
And also this line is not changed in this PR.